src/app/pages/todos/components/todo-page/todo-page.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { ITodo, TodoFacadeService } from '@shared';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-todo-page',
+  templateUrl: './todo-page.component.html',
+  styleUrls: ['./todo-page.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TodoPageComponent implements OnInit {
+  todos$: Observable<ITodo[]>;
+
+  constructor(private todoFacadeService: TodoFacadeService) {}
+
+  ngOnInit(): void {
+    this.todoFacadeService.loadTodoList();
+    this.todos$ = this.todoFacadeService.getTodoList();
```

Comment:
you're using 2 async pipes for observable which is unicasting.
Use `shareReplay({ bufferSize: 1, refCount: true })` in pipe to make it multicasting and having 1 source of observable